src/libui_sdl/libui/windows/CMakeLists.txt

```diff
@@ -80,7 +80,7 @@ endmacro()
 # notice that usp10 comes before gdi32
 # TODO prune this list
 set(_LIBUI_LIBS
-	user32 kernel32 usp10 gdi32 comctl32 uxtheme msimg32 comdlg32 d2d1 dwrite ole32 oleaut32 oleacc uuid
+	user32 kernel32 usp10 gdi32 comctl32 uxtheme msimg32 comdlg32 d2d1 dwrite ole32 oleaut32 oleacc uuid opengl32 shell32
```

Comment:
Without explicitly linking `opengl32` and `shell32`, I was running into linking errors in `libui`